SA/concept_questionnaire/concept_questionnaire.typ

```diff
@@ -0,0 +1,157 @@
+#import "../style.typ": *
+#show heading.where(level: 1): it => { text(it, size: 1.3em); v(2em); }
+#set page(footer: sa_footer((authors-short: [L. Streckeisen, J. Flepp])))
+#set text(size: 1.2em)
+
+#let concept(data) = [
+  #set page(
+    margin: (bottom: 2cm, top: 1cm, left: 1cm, right: 1cm))
+
+  #let scenario(title, content, height: 6cm) = block(
+      inset: 8pt,
+      radius: 6pt,
+      width: 100%,
+      stroke: .5pt,
+      height: height,
+      fill: rgb("F7F7F7"))[
+    === #title
+    #line(length: 100%, stroke: .5pt)
+    #content
+  ]
+
+  = #data.title
+  #v(-1em)
+
+  #let gutter = .5cm
+  #columns(2, gutter: gutter)[
+    #scenario("Simple Addition Function")[
+      #data.simple_addition
+    ]
+    #colbreak()
+    #scenario("Even Numbers from 1 to 10")[
+      #data.even_one_to_ten
+    ]
+  ]
+  #v(gutter)
+  #v(-1em)
+  #columns(2, gutter: gutter)[
+    #scenario("Product of Numbers")[
+      #data.product_of_numbers
+    ]
+    #colbreak()
+    #scenario("Map Add 5 Function")[
+      #data.map_add_5_function
+    ]
+  ]
+  #v(gutter)
+  #v(-0.8em)
+  #scenario("Count Words", height: 10cm)[
+      #data.count_words
+  ]
+]
+
+#[
+= VisualFP Concept Questionare
+
+Hi there,
+
+In the context of our SA, we are currently searching for a new way to
+visualize functional programming concepts. Before we start to flesh out our
+ideas, we would like to get some feedback on a few concepts we came up with.
+
```

Comment:
We'll then decide which concept we'll develop further based on the received feedback.

Reply:
added